src/js/reusable-charts/horizontal-bar-chart.js

```diff
@@ -56,6 +59,8 @@ export function horizontalBarChart() {
     let xLabel = initialConfiguration.xLabel;
     let barLabelLength = initialConfiguration.barLabelLength;
     let reverse = initialConfiguration.reverse;
+    let minX = initialConfiguration.minX;
+    let maxX = initialConfiguration.maxX;
```

Comment:
this could be written this 
```js
let maxX = initialConfiguration.maxX || max(data, d => d.value);
```

Reply:
I don't think you can because you don't yet know what data is - and it may change in future